python_modules/dagster/dagster/_core/definitions/schema_change_checks.py

```diff
@@ -0,0 +1,148 @@
+from typing import Dict, Mapping, Sequence, Union, cast
+
+from pydantic import BaseModel
+
+from dagster._annotations import experimental
+
+from .asset_check_result import AssetCheckResult
+from .asset_check_spec import AssetCheckSeverity, AssetCheckSpec
+from .asset_checks import AssetChecksDefinition
+from .asset_key import AssetKey, CoercibleToAssetKey
+from .assets import AssetsDefinition, SourceAsset
+from .decorators.asset_check_decorator import multi_asset_check
+from .events import AssetMaterialization
+from .metadata import TableColumn, TableMetadataEntries, TableSchema
+
+
+@experimental
+def build_column_schema_change_checks(
+    *,
+    assets: Sequence[Union[CoercibleToAssetKey, AssetsDefinition, SourceAsset]],
+    severity: AssetCheckSeverity = AssetCheckSeverity.WARN,
+) -> Sequence[AssetChecksDefinition]:
+    """Returns asset checks that pass if the the column schema of the asset's latest materialization
+    is the same as the column schema of the asset's previous materialization.
+
+    Args:
+        assets (Sequence[Union[AssetKey, str, AssetsDefinition, SourceAsset]]): The assets to create
+            asset checks for.
+        severity (AssetCheckSeverity): The severity if the check fails. Defaults to WARN.
+
+    Returns:
+        Sequence[AssetsChecksDefinition]
+    """
+    asset_keys = set()
+    for el in assets:
```

Comment:
what's el

Reply:
Short for "element". "asset" and "assets_def" didn't make sense, because different elements can be singular or plural assets.